pkgs/applications/version-management/git-and-tools/git-appraise/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "git-appraise-unstable-${version}";
+  version = "2018-02-26";
+  rev = "2414523905939525559e4b2498c5597f86193b61";
+
+  goPackagePath = "github.com/google/git-appraise";
+
+  src = fetchgit {
```

Comment:
Is there any reason to not use fetchFromGitHub here? It saves a lot of bandwidth.

Reply:
Not really, it's just how `go2nix` did it. I'll update :wink: